File: PhotosMaker/PhotosMaker/ViewController.m
//
//  ViewController.m
//  PhotosMaker
//
//  Created by Darsky on 2018/2/6.
//  Copyright © 2018年 Darsky. All rights reserved.
//

#import "ViewController.h"
#import "ZWPhotosMakerAssetViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (IBAction)didStartButtonTouch:(id)sender
{
    ZWPhotosMakerAssetViewController *viewController = nil;
    viewController = [[ZWPhotosMakerAssetViewController alloc] initWithNibName:@"ZWPhotosMakerAssetViewController"
                                                                        bundle:nil];
    [self presentViewController:[[UINavigationController alloc] initWithRootViewController:viewController]
                       animated:YES
                     completion:^{
                         
                     }];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Swift/PhotosMaker/PhotosMaker/ViewController.swift
//
//  ViewController.swift
//  PhotosMaker
//
//  Created by Darsky on 2018/2/24.
//  Copyright © 2018年 Darsky. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    @IBAction func didStartButtonTouch(_ sender: Any)
    {
        let viewController:AssetsViewController = AssetsViewController(nibName: "AssetsViewController", bundle: Bundle.main)
        self.present(UINavigationController.init(rootViewController: viewController), animated: true)
        {
            
        }
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

